### PR TITLE
Translate 50+ common pages in Italian.

### PR DESCRIPTION
**Pull request by mebeim**
Added Italian translation for 50+ pages of the `common` section:

    ansiweather apd apm apropos ar aria2c arp asar asciinema assimp astyle at atom atq atrm autoflake autojump autossh avrdude awk aws aws-s3 axel az b2sum babel badblocks banner base32 base64 basename bash bashmarks bat batch bc beanstalkd bedtools berks bg blender bmaptool boot borg bosh bower box browser-sync bundle bup bw c99 cabal calibre-server calibredb cargo case cat cd

---

Let me know if you want to keep all the commits or squash them somehow.

### CODE REVIEW

**Comment by mebeim** [comment]
Thank you guys 😁 I'm also writing my own client and it has language support hahaha coming soon I hope

**Comment by agnivade** [comment]
@waldyrious - Any comments on the PR ?

File: pages.it/common/assimp.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`assimp listext`
+
+- Elenca tutti i formati supportati per l'esportazione:
+
+`assimp listexport`
+
+- Converti un file a uno dei tipi supportati, utilizzando i parametri predefiniti:
+
+`assimp export {{file_input.stl}} {{file_output.obj}}`
```
Comment (sbrl) [inline pages.it/common/assimp.md]: Sometimes you've translated the filenames, and sometimes you haven't - as here.

Comment (mebeim) [reply pages.it/common/assimp.md]: That is indeed translated. "*Input file*" becomes "*file input*" in italian. 

The word "input" is used in Italian too (in this case as an adjective), the only difference is that the adjective in italian comes after the name. 

"*This is an input file*" → "*questo è un file input*" or "*questo è un file di input*".

**Comment by mebeim** [comment]
@sbrl 

> Have you considered contributing back to any of the existing clients to add language support?

Not that much since the desire to write my own client came first, before writing a language spec. Once done, I'll probably take a look around and contribute on others too.

File: pages.it/common/assimp.md (hunk 1 above)
Comment (sbrl) [reply pages.it/common/assimp.md]: Ah, I see! Cool :D

**Comment by sbrl** [comment]
Awesome!

If there's no activity from @waldyrious here in, say, a week or so, I'll merge this.

**Comment by agnivade** [comment]
No issues.

**Comment by sbrl** [comment]
Awesome, @mebeim! It looks like @waldyrious is rather busy, so I'll merge this PR for now.

Just open a new PR if you translate any more pages :-)

**Pull request merged**